version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "1.5.0-SNAPSHOT"
+version in ThisBuild := "1.4.1-SNAPSHOT"
```

Comment:
I could delete the commits and published artifacts for 1.4.0 and republish that version, since it was only published a few minutes ago, but I thought this was safer.